server/app.js

```diff
@@ -47,6 +62,20 @@ app.use('/courses', coursesRouter);
 app.use('/scraping', scriptRouter);
 app.use('/auth', authRouter);
 
+// Serving the frontend on wildcard. NEEDS TO REMAIN ABOVE 404 error
+if (process.env.NODE_ENV === 'production') {
+  app.get('/*', (req, res) => {
+    res.sendFile(path.join(__dirname, '..', 'frontend', 'build', 'index.html'));
+  });
+} else if (doesBuildDirExist) {
+  // checks to be able to see server endpoints in development
+  app.get('/*', (req, res) => {
+    // probably will never serve the generator files gain
+    // res.sendFile(path.join(__dirname, '..', 'frontend', 'public', 'index.html'));
```

Comment:
Maybe remove comment if it's unnecessary; not sure what "will never serve generator files again" means also